store/chat_test.go

```diff
@@ -0,0 +1,175 @@
+package store
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/pastelnetwork/go-commons/log"
+)
+
+var (
+	chatDB *ChatDB
+)
+
+func TestMain(m *testing.M) {
+	tmpDir, err := ioutil.TempDir("", "chat")
+	if err != nil {
+		panic(err)
```

Comment:
Do not use `panic` anywhere in your code! In this place you need to use `t.Fatal()`